src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -14,22 +14,12 @@
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Helper\Helper;
 use Symfony\Component\Console\Output\StreamOutput;
-use Symfony\Component\Console\Tests;
-
-require_once __DIR__.'/../ClockMock.php';
 
+/**
+ * @group clockMock
```

Comment:
i'm open for suggestions :) using a group annotation makes the implementation a lot easier by reusing a bunch of existing caching logic. It also allows filtering these specific tests when running the test suite...